src/pybind/mgr/dashboard/__init__.py

```diff
@@ -43,7 +43,5 @@ def __getattr__(self, item):
     mgr = mock.Mock()
     mgr.get_frontend_path.side_effect = lambda: os.path.abspath("./frontend/dist")
 
-    mock_ceph_modules()
-
```

Comment:
Shouldn't we remove the import?

Reply:
if you don't import `tests`, nothing will be mocked and things will break. 

you have a better idea?

Reply:
I just meant the `mock_ceph_modules`, which we no longer use.